esmvaltool/config-developer.yml

```diff
@@ -234,7 +234,7 @@ OBS:
     default: 'Tier[tier]/[dataset]'
     BSC: '[type]/[institute.lower]/[dataset.lower]/[freq_folder]/[short_name][freq_base]'
   input_file:
-    default: '[project]_[dataset]_[type]_[version]_[field]_[short_name]_*.nc'
+    default: '[project]_[dataset]_[type]_[version]_[mip]_[short_name]_*.nc'
```

Comment:
Do not change this yet, we are still using the v1 data. I will make this change in another PR once all the data are ready and synchronized to Jasmin.